src/main/java/org/opensearch/geospatial/plugin/GeospatialPlugin.java

```diff
@@ -160,7 +163,8 @@ public List<RestHandler> getRestHandlers(
             new RestUploadStatsAction(),
             new RestUploadGeoJSONAction(),
             new RestPutDatasourceHandler(clusterSettings),
-            new RestGetDatasourceHandler()
+            new RestGetDatasourceHandler(),
+            new RestUpdateDatasourceHandler()
```

Comment:
nit: What do you think about creating a new list per domain ?
ex: List for UploadGeoJosn, List of DataSource (ip2GeoJSON) and merge both list in this method. This will help to keep relevant code together.

Reply:
Sure.